assets/cart-drawer.css

```diff
@@ -19,129 +20,121 @@
 }
 .cart-drawer .cart-drawer__close {
   position: absolute;
-  top: 14px;
-  left: 13px;
+  top: 13px;
+  left: 13.5px;
   background-color: transparent;
   border: none;
-  font-size: 28px;
+  font-size: 36px;
   z-index: 1000;
 }
 [dir=ltr] .cart-drawer .cart-drawer__close {
   left: unset;
-  right: 13px;
+  right: 13.5px;
 }
 .cart-drawer .header {
   display: flex;
   padding: 16px 24px;
+  border-bottom: 0.5px solid #ADADAD;
   position: sticky;
   top: 0;
   background-color: white;
   z-index: 100;
 }
 .cart-drawer .header .cart {
   font-weight: 700;
-  font-size: 22px;
+  font-size: 26px;
   display: flex;
-  gap: 12px;
+  gap: 6px;
   align-items: center;
 }
 .cart-drawer .header .cart span {
-  font-weight: 400;
-  font-size: 15px;
-  color: #9B9B9B;
+  font-weight: 600;
+  font-size: 12px;
+  background-color: var(--yc-primary-color);
+  height: 18px;
+  width: 18px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  color: #fff;
 }
 .cart-drawer .footer {
-  background: white;
+  background-color: white;
   border-top: 1px solid #EBEBEB;
   position: absolute;
   bottom: 0;
-  padding: 26px 24px;
+  padding: 26px 24px 32px;
   width: 100%;
 }
 .cart-drawer .footer .price-wrapper {
   display: flex;
   justify-content: space-between;
 }
 .cart-drawer .footer .price-wrapper .total-price {
-  font-weight: 400;
+  font-weight: 700;
   font-size: 16px;
 }
 .cart-drawer .footer .price-wrapper .currency-value {
   font-weight: 700;
-  font-size: 16px;
+  font-size: 18px;
+  color: var(--yc-primary-color);
 }
 .cart-drawer .footer .price-wrapper .currency-code {
   font-weight: 700;
+  color: var(--yc-primary-color);
 }
 .cart-drawer .footer a {
-  margin-top: 16px;
-}
-.cart-drawer .footer .cart-action {
-  border: 1px solid var(--yc-primary-color);
-  padding: 0 2rem;
-  transition: all 0.15s ease-in-out 0s;
-  letter-spacing: 1px;
-  border-radius: 4px;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  height: 43px;
+  margin-top: 24px;
 }
 .cart-drawer .cart-drawer__content {
-  max-height: calc(100% - 196px);
+  max-height: calc(100% - 147px);
   overflow-y: auto;
 }
 .cart-drawer ul {
-  margin: 20px 24px 0;
-}
-.cart-drawer ul .cart-item {
-  margin-bottom: 24px;
+  margin: 0 24px;
 }
 .cart-drawer ul .cart-item img {
-  width: auto;
+  width: 82px;
   height: 106px;
```

Comment:
I noticed that the image gets skewed if it's not square, can you please add an `object-fit: contain`

Reply:
in that case we will go with `object-fit: cover ` so that we can make sure all images fit the box preserving its aspect-ratio.

Reply:
Sure, that's good